### ISSUE TITLE
Issue with installer timezones

### ISSUE THREAD

**Issue by staticfrost** [anchorcms/anchor-cms#1145]
### Summary

There is an issue with the timezone selector.

It's not showing correct time zones.

Brisbane Australia is +10 GMT / UTC... has no daylight savings time.

Yet it list it as +9 GMT.

http://www.timetemperature.com/australia/brisbane_time_zone.shtml
### Expected Behaviour

+10 incorrect timezone

### Actual Behaviour

+10 is the correct timezone,
if this is wrong, i'd assume alot of the other ones are incorrect too.
### Context details (if applicable)

- Anchor version: .Powered by Anchor, version 0.12.1
- Server setup: shared cpanel host
- URL

**Comment by TheBrenny** [comment]
This issue has already been fixed, please update to the [latest version (0.12.1)](https://github.com/anchorcms/anchor-cms/releases).

**Comment by staticfrost** [comment]
That is the version I am using.

**Comment by TheBrenny** [comment]
In your context details you mention that your anchor version is .93, and I assumed you meant 0.9.3? Can you please confirm this?

**Comment by Gerrit0** [comment]
I fixed this a while back: https://github.com/anchorcms/anchor-cms/pull/1101

That's a really old version of anchor...

**Comment by staticfrost** [comment]
sorry, as per admin console - Powered by Anchor, version 0.12.1

**Comment by Gerrit0** [comment]
Ahh, I'm guessing you are on migration 93. 

This is a visual only glitch for the record. If you select the timezone, it should still correctly use the timezone.

**Comment by staticfrost** [comment]
sorry fresh install i'm not sure why i thought it was 93.

How can i confirm timezones easily enough?

**Comment by Gerrit0** [comment]
I believe it should be the option labeled `GMT+2 Europe/Berlin`, as that will choose the PHP timezone of `Europe/Berlin`, I've moved on from Anchor however so really can't say for sure. 

Looks like I might have to return briefly as this will now bug me to no end :)

**Comment by nitwhiz** [comment]
Could it be that this is some daylight saving time stuff?
I referred to [this map](http://www.eclipsegeeks.com/communities/3/004/009/983/113//images/4569743367.jpg), where it says UTC=GMT and London **has to be** GMT+0.
Berlin always was GMT+1 in user interfaces i saw.
Could be i got something wrong there and UTC is handled differently when it comes to DST though, as timeanddate.com suggests.

**Comment by Gerrit0** [comment]
Yep! That would do it. Do you have a recommendation for how the list should display? Should we just completely ignore DST when generating the timezone list?

**Comment by nitwhiz** [comment]
UTC ignores DST, so my suggestion would be to not display timezones as UTC, not as GMT.

The question is, what do users expect?
- Windows Time Settings use UTC
- Google Calendar uses GMT
- Twitter uses GMT
- Wordpress seems to use UTC (only checked some manual, don't have an instance up)
- Joomla seems to use UTC (see Wordpress)
- _Personally_ I'd be like _oh wtf_ when I see UTC, maybe google it and say _oooh_, or I'd just know I have to choose "+1 Berlin/Amsterdam/...". Can't really tell what's the best UX here.

I guess the mixture of DST and GMT (which we know as +0) makes it weird at first because we think London has to be +0 too then.^^

**TL;DR:**
As long as the city names are kept, UTC would be the best UX imho.
